examples/tabs/palette/demo/@misk/core/features/Navbar/processNavbarItems.d.ts

```diff
@@ -1,3 +1,3 @@
 /// <reference types="react" />
-import { Environment } from "@misk/common";
+import { Environment } from "../../utilities";
```

Comment:
out of curiosity, are we digging ourselves into a hole using relative paths here?

Reply:
All code internal to a package is traditionally in JSland referenced by relative path. There are some Webpack configuration we could use to alias to a specific directory within the package.

- https://webpack.js.org/configuration/resolve/
- http://xabikos.com/2015/10/03/Webpack-aliases-and-relative-paths/